readme.md

```diff
@@ -38,10 +38,95 @@ Default: `process.cwd()`
 
 The path to where the `package.json` file should be written or its directory.
 
+#### data
+
+Type `object`
+
+JSON data to write to the `package.json` file.
+
+#### options
+
+Type: `object`
+
+See [Options](#options-4).
+
+### updatePackage(path?, data, options?)
+
+Returns a `Promise` that resolves when the `package.json` file has been updated.
+
+### updatePackageSync(path?, data, options?)
+
+#### path
+
+Type: `string`\
+Default: `process.cwd()`
+
+The path to where the `package.json` file should be written or its directory.
+
+#### data
+
+Type `object`
+
+JSON data to write to the `package.json` file. If the file already exists, existing fields will be merged with the values in `data`.
+
 #### options
 
 Type: `object`
 
+See [Options](#options-4).
+
+### addPackageDependencies(path?, dependencies, options?)
+
+Returns a `Promise` that resolves when the `package.json` file has been written.
```

Comment:
It would be useful to include the code examples from the TS types for each of these methods.

https://github.com/sindresorhus/write-pkg/pull/21/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R137-R147

Reply:
Added.